packages/x-data-grid/src/locales/frFR.ts

```diff
@@ -39,9 +39,9 @@ const frFRGrid: Partial<GridLocaleText> = {
   toolbarExportExcel: 'Télécharger pour Excel',
 
   // Columns management text
-  // columnsManagementSearchTitle: 'Search',
-  // columnsManagementNoColumns: 'No columns',
-  // columnsManagementShowHideAllText: 'Show/Hide All',
+  columnsManagementSearchTitle: 'Rechercher',
+  columnsManagementNoColumns: 'Pas de colonnes',
+  columnsManagementShowHideAllText: 'Aficher/masquer toutes',
```

Comment:
```suggestion
  columnsManagementShowHideAllText: 'Afficher/masquer toutes',
```

https://dictionnaire.lerobert.com/definition/afficher